src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -2117,20 +2117,6 @@ public void storeOpenLastFilesOnStartup(boolean openLastFilesOnStartup) {
         putBoolean(OPEN_LAST_EDITED, openLastFilesOnStartup);
     }
 
-    @Override
-    public NewLineSeparator getNewLineSeparator() {
-        return NewLineSeparator.parse(get(NEWLINE));
-    }
-
-    @Override
-    public void storeNewLineSeparator(NewLineSeparator newLineSeparator) {
-        String escapeChars = newLineSeparator.toString();
-        put(NEWLINE, escapeChars);
```

Comment:
Is there a reason you remove this from the preferences?

Reply:
Yes, there is:

> A bib file should keep the line break style. When collaborating on a network drive with different operating systems, the line breaks surely differ from the linebreaks in local files or in a drop box. Thus, the linebreaks of the .bib file itself should be respected. This is implemented in this PR. This fixes sourceforge.net/p/jabref/feature-requests/857. (See also #4877 (comment)).

Since the bib file **keeps** the line ending, there is no need to configure one.

The default line ending of a new bib file is OS dependent. 

In other words: My aim here was to reduce configuration options, too.

In case we want to bring back the option, I would include it in the library preferences tab, because the setting is library dependent.